data/muffinhunt/advancements/refunded.json

```diff
@@ -0,0 +1,33 @@
+{
+    "display": {
+      "icon": {
+        "item": "minecraft:gold_ingot"
+      },
+      "title": "Refunded",
+      "description": "Get refunded by a piglin",
+      "frame": "task",
+      "show_toast": true,
+      "announce_to_chat": true,
+      "hidden": false
+    },
+    "parent": "minecraft:story/enter_the_nether",
+    "criteria": {
+      "requirement": {
+        "trigger": "minecraft:thrown_item_picked_up_by_player",
```

Comment:
Spyglass, why are you not accepting that `thrown_item_picked_up_by_player` exists...